### PR TITLE
Added a link to documentation webpage in notebook time_series_forcast

### PR DESCRIPTION
**Pull request by jingdong00**
## Why are these changes needed?

I've added links to the AutoML class docs and the optimization metric docs to the tutorial notebook.

## Related issue number
This is related to Issue #787 

## Checks

- [x] I've used [pre-commit](https://microsoft.github.io/FLAML/docs/Contribute#pre-commit) to lint the changes in this PR, or I've made sure [lint with flake8](https://github.com/microsoft/FLAML/blob/816a82a1155b4de4705b21a615ccdff67c6da379/.github/workflows/python-package.yml#L54-L59) output is two 0s.
- [NA] I've included any doc changes needed for https://microsoft.github.io/FLAML/. See https://microsoft.github.io/FLAML/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by jingdong00** [comment]
@microsoft-github-policy-service agree [company=""]

**Comment by jingdong00** [comment]
@microsoft-github-policy-service agree

File: notebook/automl_time_series_forecast.ipynb [hunk 1]
```diff
@@ -124,6 +123,13 @@
     "automl = AutoML()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The AutoML class constructor takes a list of user-specified setting for fitting and prediction. A comprehensive list of setting options available can be found here [List of setting options](https://microsoft.github.io/FLAML/docs/reference/automl/#automl-objects). In particular, the fit function usually needs users to specifiy a metric for validation. A list of built-in optimization metric available can be found here [List of built-in optimization metric](https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML/#optimization-metric)"
```
Comment (sonichi) [inline notebook/automl_time_series_forecast.ipynb:130]:
```suggestion
    "The AutoML class constructor takes a list of user-specified setting for fitting and prediction. A comprehensive list of setting options available can be found here [List of setting options](https://microsoft.github.io/FLAML/docs/reference/automl/#automl-objects). In particular, users may want to specify a metric for optimization. A list of built-in optimization metrics available (as well as how to customize metrics) can be found at [here](https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML/#optimization-metric)"
```

**Review by skzhang1 — APPROVED** [review]

File: notebook/automl_time_series_forecast.ipynb [hunk 2]
```diff
@@ -1857,9 +5866,26 @@
...
+    },
+    {
+     "ename": "",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31mCanceled future for execute_request message before replies were done"
+     ]
+    }
+   ],
```
Comment (sonichi) [inline notebook/automl_time_series_forecast.ipynb:5888]: This cell failed

Comment (jingdong00) [reply notebook/automl_time_series_forecast.ipynb:5888]: I'll look into this tomorrow. It's seems that the test will only fail with a certain built of MacOS?

Comment (skzhang1) [reply notebook/automl_time_series_forecast.ipynb:5888]: @jingdong00 Thanks! It fails many times when I use macOS. Can you save the output of every cell before pushing?

**Comment by jingdong00** [comment]
Of course. Lemme look into this

File: notebook/automl_time_series_forecast.ipynb (hunk 2 above)
Comment (jingdong00) [reply notebook/automl_time_series_forecast.ipynb]: @skzhang1  I've added all the cell output:). However, I do find the notebook to fail under macOS (but work with Linux)

**Comment by skzhang1** [comment]
It looks good to me.

**Review by skzhang1 — APPROVED** [review]

**Review by qingyun-wu — APPROVED** [review]

**Pull request merged**